libs/designsystem/src/lib/components/page/page.component.spec.ts

```diff
@@ -111,6 +111,40 @@ describe('PageComponent', () => {
     await TestHelper.whenReady(ionContent);
   });
 
+  describe('should show the toolbar with the correct height', () => {
+    it('when on ios-phone without top safe-area', async () => {
+      ionToolbar.style.setProperty('--kirby-safe-area-top', '0px');
+      await TestHelper.resizeTestWindow(TestHelper.screensize.phone);
+
+      expect(ionToolbar).toHaveComputedStyle({ height: size('xxxl') });
+    });
```

Comment:
Consider swapping the texts around here, so it reads more like a real sentence, with 'it should xyz'. 
```
describe('toolbar with dynamic height')
it('should be shown correctly on ios-phone without top-safe-area')
```

Reply:
Done